dotCMS/src/main/java/com/dotcms/util/SecurityUtils.java

```diff
@@ -283,5 +289,18 @@ protected List<String> loadIgnoreHosts() {
     }
     return IGNORE_REFERER_FOR_HOSTS;
   }
-  
+
+
+  /**
+   * Validate if the fileName is secure and valid
+   * @param fileName  {@link String}
+   */
+  public void validateFileName (final String fileName) {
```

Comment:
We already have methods like this.  I would use them.

Reply:
specifically, `FileUtil.sanitizeFileName(xxxx)`

Reply:
yeah, I will include the sanitize but the validation throws an exception rather than escape the file name